views/components/form-fields/macros.njk

```diff
@@ -101,18 +101,30 @@
 {% endmacro %}
 
 {% macro inputTextarea(field, fieldCopy) %}
-    <label class="ff-label" for="field-{{ field.name }}">
-        {{ fieldLabelText(field, fieldCopy) }}
-    </label>
-    {{ fieldHelpText(field, fieldCopy) }}
-    <textarea
-        class="ff-textarea"
-        id="field-{{ field.name }}"
-        name="{{ field.name }}"
-        {% if field.isRequired %}required aria-required="true"{% endif %}
-        {% if field.rows %}rows="{{ field.rows }}"{% endif %}
-        {% if field.attributes %}{% for attr, value in field.attributes %} {{ attr }}="{{ value }}"{% endfor %}{% endif %}
-    >{{ field.value }}</textarea>
+    {% set settings = field.settings %}
+    {% if settings.showWordCount %}<div class="js-character-count">{% endif %}
+        <label class="ff-label" for="field-{{ field.name }}">
```

Comment:
Minor thing but this whole block needs to be outdented as it looks a bit like it's inside the `{% if %}` block above.